simulator/docker/docker-compose.yml

```diff
@@ -28,11 +28,13 @@ services:
     image: hedera-block-simulator:latest
     networks:
       - block-node_default
+    env_file:
+      - .env
     environment:
-      - GRPC_SERVER_ADDRESS=block-node-server
-      - BLOCK_STREAM_SIMULATOR_MODE=CONSUMER
-      - PROMETHEUS_ENDPOINT_ENABLED=true
-      - PROMETHEUS_ENDPOINT_PORT_NUMBER=9997
+      - GRPC_SERVER_ADDRESS=${GRPC_SERVER_ADDRESS}
+      - BLOCK_STREAM_SIMULATOR_MODE=${CONSUMER_BLOCK_STREAM_SIMULATOR_MODE}
+      - PROMETHEUS_ENDPOINT_ENABLED=${PROMETHEUS_ENDPOINT_ENABLED}
+      - PROMETHEUS_ENDPOINT_PORT_NUMBER=${CONSUMER_PROMETHEUS_ENDPOINT_PORT_NUMBER}
```

Comment:
Do we need the `environment` since now we have the `env_file`? One will take precedence over the other, I do not think we should be using both.

Reply:
Usually no. But in our case that's the case. Meaning we need them, but not all. Reason is because we are starting the same container in twice with different configuration, so it needs to now, which variable to map. Pushed a commit where I remove the unneded.